code/controllers/subsystem/religion.dm

```diff
@@ -3,10 +3,8 @@ SUBSYSTEM_DEF(religion)
 	init_order = 19
 	flags = SS_NO_FIRE|SS_NO_INIT
 
-	var/bible_deity_name
-	var/Bible_icon_state
-	var/Bible_item_state
-	var/Bible_name
-	var/Bible_deity_name
-
-	var/holy_weapon
+	var/religion
+	var/deity
+	var/bible_name
+	var/list/bible_icons = list("icon_state", "item_state")
```

Comment:
yea you are right, just a stupid idea i had at the time(that i thought it was good) and kinda regretted halfway through